src/extension.ts

```diff
@@ -492,14 +485,21 @@ function registerCommands(context: VSCode.ExtensionContext) {
     showLogOutput();
   }));
 
-  context.subscriptions.push(VSCode.commands.registerCommand(Commands.INSTALL_MANAGED_JRE, installManagedJre));
-
   context.subscriptions.push(
-    VSCode.commands.registerCommand(Commands.HIDE_HOTSPOT, async () => {
-      await hideSecurityHotspot(hotspotsTreeDataProvider);
-      updateSonarLintViewContainerBadge();
+    VSCode.commands.registerCommand('SonarLint.NewCodeDefinition.Enable', () => {
+      VSCode.workspace.getConfiguration('sonarlint')
+              .update('focusOnNewCode', true, VSCode.ConfigurationTarget.Global);
+      NewCodeDefinitionService.instance.updateNewCodeStatusBarItem(null);
```

Comment:
IMHO, this should be a call to `NewCodeDefinitionService.instance.updateFocusOnNewCodeState()` - and it may not even be needed, since it is called by the configuration listener (above in this file).